public/app/features/plugins/plugin_loader.ts

```diff
@@ -43,17 +43,28 @@ import * as d3 from 'd3';
 import * as emotion from 'emotion';
 import * as grafanaData from '@grafana/data';
 import * as grafanaUIraw from '@grafana/ui';
+import * as grafanaUIEditors from '@grafana/ui/src/editors';
 import * as grafanaRuntime from '@grafana/runtime';
 
 // Help the 6.4 to 6.5 migration
 // The base classes were moved from @grafana/ui to @grafana/data
 // This exposes the same classes on both import paths
-const grafanaUI = grafanaUIraw as any;
+let grafanaUI = grafanaUIraw as any;
 grafanaUI.PanelPlugin = grafanaData.PanelPlugin;
 grafanaUI.DataSourcePlugin = grafanaData.DataSourcePlugin;
 grafanaUI.AppPlugin = grafanaData.AppPlugin;
 grafanaUI.DataSourceApi = grafanaData.DataSourceApi;
 
+/**
+ * 6.XXX introduces namespaced exports from @grafana/ui
+ * This makes sure that exports previously accessible via direct imports from @grafana/ui are still available
+ * via global namespace
+ */
+grafanaUI = {
```

Comment:
@torkelo @ryantxu adding namespaced exports to global grafana/ui namespace for backward compatibility